contrib/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:14.04
```

Comment:
I had some trouble getting Node to run on debian last time I tried, so I defaulted to what I knew that worked well.

I will make a note to change it to something else, like Debian, before we upload it to Docker hub.